tests/test_automation_snippets.py

```
@@ -0,0 +1,102 @@
+        state_env="$HOME/.local/state",
+    )
+    assert snippet.startswith("*/20 * * * * ")
+    assert f"cd {tmp_path}" in snippet
+    assert log_path in snippet
+    assert "--html" in snippet
+    assert "--collapse-threshold 20" in snippet
+
+
+def test_describe_targets_roundtrip():
+    data = describe_targets()
+    assert "codex" in data and "claude-code" in data and "drive-sync" in data and "gemini-render" in data
```

Ruff (0.15.9)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)